R/rosters_players.R

```diff
@@ -0,0 +1,29 @@
+#' Process `load_players()` as root of players dataframe
```

Comment:
We should come back to the players <-> rosters logic once we're ready to cut over the players data to production, but I'm going to consider it outside of the scope of this PR